docs/src/main/asciidoc/websockets-next-reference.adoc

```diff
@@ -0,0 +1,546 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= WebSockets Next Extension Reference Guide
+:extension-status: preview
+include::_attributes.adoc[]
+:numbered:
+:sectnums:
+:categories: web
+:topics: web,websockets
+:extensions: io.quarkus:quarkus-websockets-next
+
```

Comment:
I think that both guides should start with a section like "Exprimental status"  that would explain that the API may change in the future. WDYT?

Reply:
Done.